docs/source/user-guide/tasks/build-packages/define-metadata.rst

```diff
@@ -618,22 +618,42 @@ package may not be testable, because the hashes will differ.
 
 .. _run_exports:
 
-Pin downstream
---------------
+Export runtime requirements
+---------------------------
 
-It is often useful to require a particular runtime library to be present at
-runtime when a particular library or package is used at build time. For example,
-if you use a C++ compiler to build a package with dynamic linkage, then it is
-likely that you also need to include the C++ runtime package corresponding to
-that compiler as a runtime requirement. Generally, these imposed pinnings should
-be added to the tool (compiler) or library (jpeg, bzip2, and so on) used at build
-time, rather than to the package using those tools or libraries.
+Some build or host :ref:`requirements` will impose a runtime requirement.
+Most commonly this is true for shared libraries (e.g. libpng), which are
+required for linking at build time, and for resolving the link at run time.
+With ``run_exports`` (new in conda-build 3) such a runtime requirement can be
+implicitly added by host requirements (e.g. libpng exports libpng), and with
+``run_exports/strong`` even by build requirements (e.g. gcc exports libgcc).
 
 .. code-block:: yaml
 
+   # meta.yaml of libpng
    build:
      run_exports:
-       - libstdc++
+       - libpng
+
```

Comment:
I think it is worthwhile to add more description in between examples of weak and strong run exports.

I'll do that and push to your PR here.